exp/api/v1alpha4/azuremachinepool_conversion.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v1alpha4
 
 import (
+	machineryConversion "k8s.io/apimachinery/pkg/conversion"
```

Comment:
```suggestion
	apiMachineryConversion "k8s.io/apimachinery/pkg/conversion"
```
Nit: for consistency with import name in other conversions.